runtime/common/src/chain_ext.rs

```diff
@@ -1,300 +0,0 @@
-use crate::*;
```

Comment:
Why is this unrelated file removed here?

Reply:
I think it's because of [this](https://github.com/pendulum-chain/pendulum/pull/437) already merged PR

Reply:
Ah, yes, sure. Because of [this comment](https://github.com/pendulum-chain/pendulum/pull/437#issuecomment-2031746174).